### DISCUSSION TITLE
How do I load my quarto params in my R environment on VS code?

### DISCUSSION THREAD

**Discussion by giovanniromeo-annalect** [quarto-dev/quarto-cli#8836] [Q&A]
### Description

I'm encountering this problem: the params are not loaded in the R environment when using the terminal, but only when compiling the file. This is working just fine on R studio. Do you know what R studio is doing that vs code is not doing to properly load params in the environment?
Thank you for the help

````qmd
---
title: "Untitled"
format: html
params:
  test: "This is a test"
---

```{r}
print(params$test)
```
````

**Comment by mcanouil** [comment]
That's not a Quarto issue.
Quarto VSCode extension is using the R VSCode extension which can't do that.
See
- https://github.com/REditorSupport/vscode-R/issues/504
